Backend/KFC-WebAPI/Scripts/index.d.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * @fileoverview This file only declares the public portions of the API.
+ * It should not define internal pieces such as utils or modifier details.
+ *
+ * Original definitions by: edcarroll <https://github.com/edcarroll>, ggray <https://github.com/giladgray>, rhysd <https://rhysd.github.io>, joscha <https://github.com/joscha>, seckardt <https://github.com/seckardt>, marcfallows <https://github.com/marcfallows>
```

Comment:
This is not supposed to be in here

Reply:
KFC-WebAPI/Scripts

Reply:
I merged the branch 2 days ago. I don't have anymore

Reply:
with the bug branch